### ISSUE TITLE
Issue with docs.zowe.org/stable/getting-started/zowe_v2_faq

### ISSUE THREAD

**Issue by armstro** [zowe/docs-site#3236]
## Description

Question - Does Zowe require SDSF? 

## Pages to Update

FAQ
## Expected behavior

There is not a hard dependency on SDSF.  If it’s there some of the configuration code that tries to auto-detect parameter defaults will take advantage of it, however if the LPAR doesn’t have SDSF (perhaps it has EJES or the Broadcom CA Sysview) then the sysprog has to fill in the values by hand. 

## Additional context

Text provided by @Joe-Winchester - to be confirmed

**Comment by samanthasusu** [comment]
Hi Bruce, thanks for creating the issue. If it's confirmed, I will add it to the FAQ section.

**Comment by Joe-Winchester** [comment]
@1000TurquoisePogs,  can you confirm that there isn't a hard dependency on SDSF ?  It's possible there is a regression that slid in, but my understanding is that the SDSF code is optionally invoked.   
Some customers will have Sysview and EJES and I believe then the 'sdsf.jar' won't be present on their LPARs.

**Comment by 1000TurquoisePogs** [comment]
I can speak for myself that I didn't add SDSF code.
Can I ask questions for more context?
1) What does this have to do with docs.zowe.org/stable/getting-started/zowe_v2_faq ? I don't see SDSF mentioned on that page.
2) Did someone run into an issue where it tried to use SDSF or said that SDSF was required?

**Comment by armstro** [comment]
Yes customer came in with a question on SDSF as pre-req? I could not easily find the answer (other than it not be listed in the System Requirements) so FAQ seemed like a place to put it. I asked in the ZAC and it was suggested to open an issue.

**Comment by 1000TurquoisePogs** [comment]
I think you're having trouble finding SDSF in the doc simply because it isn't a prereq, like many other things.
The only use of SDSF that I know of is that `zwe start` and `zwe start` use opercmd which is a feature of SDSF. But, thats neither the only way to start zowe nor the popular way... people just start the Zowe STC in the tool of their choice. Only, when its not SDSF, I don't have advice because that's what I use. But the way the STC is started is ordinary, so no guidance needed?

**Comment by armstro** [comment]
Let me go back to the customer and see why they think it was a prereq - here is the text of the email .

..they are not using SDSF and they found out that the newer version of Zowe is depending on that? Can you shine a bit of light on that? How dependent is that, can they bypass SDSF at all or is it a fixed requirement?

It also seems that sometime in the past this dependency started. They still have an older version of Zowe (not sure which level) and they didn’t had any problems but a recent version does throw up some errors?

**Comment by anaxceron** [comment]
@armstro @1000TurquoisePogs @Joe-Winchester -- do we need an FAQ on this, then?
